.circleci/run_tflint.sh

```diff
@@ -18,6 +18,9 @@ for a in $(ls examples/stage);
         rm -rf .terraform;
         cp main.tf main.tf.bck;
         cp ../../../.circleci/sample/main.tf.sample main.tf;
+        echo "terraform format check"
+        terraform fmt -check -recursive
```

Comment:
included terraform fmt check as part of circleci but later we can look an option of using GitHub Actions and move away from circleci - @tom-butler